lib/rex/exceptions.rb

```diff
@@ -255,11 +269,10 @@ def to_s
 #
 ##
 class AddressInUse < ConnectionError
-  include SocketError
-  include HostCommunicationError
```

Comment:
Are there any more details on these lines being deleted? 👀 

Was this needed for folk to match particular exceptions classes with either `e.is_a?(HostCommunicationError)` or with `rescue HostCommunicationError`?

Reply:
I see it's included via `ConnectionError` anyway, LGTM :+1: